vendor/vendor.json

```diff
@@ -286,10 +286,10 @@
 			"revisionTime": "2016-12-09T21:02:51Z"
 		},
 		{
-			"checksumSHA1": "O7k9QSM9y5D/nnDFZuDrWIzuAfY=",
+			"checksumSHA1": "cYBWzzaRbkBI848bMlYapMC4+Cs=",
 			"path": "github.com/keybase/go-codec/codec",
-			"revision": "8a39c1d1f00af239dd650a292922fa4f46a39fa2",
-			"revisionTime": "2016-04-05T00:08:33Z"
+			"revision": "109532272978615a09285fb7bd1c0c7ac6be9ac5",
```

Comment:
I don't see this commit in keybase/go-codec, which hasn't been updated since March.  Can you push it there first?

Reply:
This commit is the branch `master-2018-04-24` in keybase/go-codec. I'd rather merge that branch into master last, since vendors are more easily rolled back. But I'll be doing all the vendors and merging close to the same time, so unless you feel strongly I'd rather not merge it first now.

Reply:
I only feel strongly that the commit in the repo matches a master branch commit.  But if you promise to update it everywhere right after you merge, I'm ok with it temporarily being wrong here.

Though I don't really understand why the vendors get harder to roll back once the source branch is merged.  Seems like a revert works just as well no matter what happens at the source.

Reply:
Yeap, I promise to do that!

I mean that vendors are more easily rolled back than merging a branch into go-codec's master. If it turns out the new version is buggy, it's possible that someone might pull saltpack (and thus go-codec) in the window when it's buggy and then forget to update. We control the vendored versions completely, so it's easier to manage.